src/js/translations/en-US.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@rcorral not sure if there is anything I can add because the [documentation](https://github.com/yahoo/react-intl/wiki/Components#string-formatting-components) in the React-intl repo is pretty clear.

Now about the IDs scheme I was going to start another PR for it since this one is only the code for the first experimental work.